### DISCUSSION TITLE
Will this project be archived due to Spring GraphQL?

### DISCUSSION THREAD

**Discussion by Sam-Kruglov** [graphql-java-kickstart/graphql-spring-boot#675] [Q&A]
The Spring team is going to maintain a [new project](https://github.com/spring-projects/spring-graphql), so is this one going to be deprecated?

See https://spring.io/blog/2021/07/06/hello-spring-graphql

**Comment by oliemansm** [comment]
We encourage a Spring maintained library for GraphQL support of course. At first glance though that library is still in it's early stages and is missing a lot of features that this project is providing. That's also not very strange since it's based on the GraphQL Java Spring project from the GraphQL Java guys.

To answer your question: that's a bit too early to tell. If at some point most important features of this project are supported there as well then this project will indeed become obsolete and we'll deprecate it.
